### PR TITLE
[docs] - add a guide for scheduling assets

### PR DESCRIPTION
**Pull request by slopp**
### Summary & Motivation

Declarative scheduling is a new concept that is very powerful. This tutorial guides the reader through a series of code samples that build up to a fully declarative schedule using freshness policies, helping them gain intuition of how this new approach compares to traditional cron scheduling.

### How I Tested These Changes
- code samples run
- tutorial renders
- links work

### CODE REVIEW

File: docs/content/_navigation.json [hunk 1]
```diff
@@ -50,6 +50,10 @@
           {
             "title": "Testing Assets",
             "path": "/tutorial/assets/testing-assets"
+          },
+          {
+            "title": "Scheduling Assets",
+            "path": "/tutorial/assets/scheduling-assets"
```
Comment (yuhan) [inline docs/content/_navigation.json:56]: the other tutorials are more sequential, i.e. they are step-by-step instructions that build on top of the previous page, while this one feels like a guide. should we move this tutorial to guides here https://docs.dagster.io/guides/general, and add a link to the guide here: https://dagster-git-docs-lopp-freshness-tutorial-elementl.vercel.app/tutorial/assets/testing-assets#conclusion?

Comment (yuhan) [reply docs/content/_navigation.json:56]: when it's a guide, it's ok to use arbitrary a, b, c as asset names. if we decide to keep this as the last step in the tutorial, naming assets something more meaningful would make more sense.

**Review by erinkcochran87 — CHANGES_REQUESTED** [review]
Back to you! Left comments on the Vercel preview. LMK if you'd like any help with this!

**Comment by slopp** [comment]
@erinkcochran87  thanks for the thoughtful review and fixing all my embarrassing grammar mistakes. I believe I addressed all the comments. I think the only remaining items are re: images

I double checked the majority of the images have the correct asset highlighted so I don't think further highlighting is critical

You mentioned one screenshot that could use some re-formatting though, would you be able to assist with that one?

**Review by erinkcochran87 — APPROVED** [review]

**Pull request merged**